azurerm/resource_arm_network_interface.go

```diff
@@ -357,15 +368,19 @@ func resourceArmNetworkInterfaceRead(d *schema.ResourceData, meta interface{}) e
 	if props.IPConfigurations != nil && len(*props.IPConfigurations) > 0 {
 		configs := *props.IPConfigurations
 
-		if configs[0].InterfaceIPConfigurationPropertiesFormat != nil {
-			privateIPAddress := configs[0].InterfaceIPConfigurationPropertiesFormat.PrivateIPAddress
-			d.Set("private_ip_address", *privateIPAddress)
+		if ipProps := configs[0].InterfaceIPConfigurationPropertiesFormat; ipProps != nil {
+			if privateIPAddress := ipProps.PrivateIPAddress; privateIPAddress != nil {
+				d.Set("private_ip_address", *privateIPAddress)
+			}
+			d.Set("private_ip_address_version", string(ipProps.PrivateIPAddressVersion))
```

Comment:
This does not exist in the schema

Reply:
@katbyte Add `private_ip_address_version` to the schema here? https://github.com/literatesnow/terraform-provider-azurerm/blob/b44b8b7275cbda465392b08df85c89a3c0fb04ab/azurerm/resource_arm_network_interface.go#L213

I'd say you'd know better if it's computed or not :)